source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -15,80 +15,87 @@ Metrics and Alerts
 MinIO publishes cluster and node metrics using the :prometheus-docs:`Prometheus Data Model <concepts/data_model/>`.
 You can use any scraping tool to pull metrics data from MinIO for further analysis and alerting.
 
-MinIO provides scraping endpoints for the following metric groups:
+Starting with MinIO Server :minio-release:`RELEASE.2024-07-15T19-02-30Z` and MinIO Client :mc-release:`RELEASE.2024-07-11T18-01-28Z`, metrics version 3 replaces the deprecated :ref:`metrics version 2 <minio-metrics-v2>`.
```

Comment:
`:mc-release:` shows the full URL while `:minio-release:` gives it nice text. Maybe we can make them both do the same?

Reply:
Nevermind, just did the thing already.